pkg/cache/cache_test.go

```diff
@@ -394,11 +395,13 @@ func CacheTest(createCacheFunc func(config *rest.Config, opts cache.Options) (ca
 					Expect(err).NotTo(HaveOccurred())
 
 					By("running the cache and waiting for it to sync")
+					ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Why do we need a new `context` here and in the two tests below?

Reply:
Fixed